manifests/repo/apt.pp

```diff
@@ -55,36 +57,31 @@
   }
 
   # the Gluster repo only supports x86_64 (amd64) and arm64. The Ubuntu PPA also supports armhf and arm64.
-  case $::operatingsystem {
+  case $facts['os']['name'] {
     'Debian': {
-      case $::lsbdistcodename {
-        'jessie', 'stretch':  {
-          $arch = $::architecture ? {
-            'amd64'      => 'amd64',
-            'arm64'      => 'arm64',
-            default      => false,
-          }
-          if versioncmp($release, '3.12') < 0 {
-            $repo_url  = "https://download.gluster.org/pub/gluster/glusterfs/${release}/LATEST/Debian/${::lsbdistcodename}/apt/"
-          } else {
-            $repo_url  = "https://download.gluster.org/pub/gluster/glusterfs/${release}/LATEST/Debian/${::lsbdistcodename}/${arch}/apt/"
-          }
-        }
+      $arch = $facts['architecture'] ? {
+        'amd64' => 'amd64',
+        'arm64' => 'arm64',
+        default => false,
+      }
+      if versioncmp($release, '3.12') < 0 {
```

Comment:
You can drop this since 4.0 and older are no longer on the mirrors

Reply:
dropeed it